src/mon/MonClient.cc

```diff
@@ -770,11 +769,24 @@ void MonClient::tick()
 
       send_log();
     }
+
+    _un_backoff();
```

Comment:
At this point, we may have just called `_reopen_session()`, and increased `reopen_interval_multiplier` as we started hunting. And in turn we're likely going to reduce the backoff, this tick around, to a value that may be smaller than the one set in `_start_hunting()`.  It may not make a big difference in the end, but seems a distortion of the backoff's purpose. OTOH, should this line be moved to just after/before the `send_keepalive()`, I think the purpose would be maintained, allowing `_start_hunting()` to build on the current value instead of being overridden.